config/routes.rb

```diff
@@ -172,6 +162,16 @@
           get :post_validation_requests, on: :collection
         end
 
+        resource :legislation, only: %i[show update]
+
+        concern :cancel_validation_requests do
+          member do
+            get :cancel_confirmation
+
+            patch :cancel
+          end
+        end
```

Comment:
Not for this PR but it should be a nested singleton resource like this:

``` ruby
concern :cancellable do
  resource :cancellation, only: %i[new show create]
end
```

The `CancellationsController` would still work with the validation request model, i.e. we wouldn't add a `Cancellation` model.